app/controllers/users/sessions_controller.rb

```diff
@@ -9,9 +9,13 @@ class Users::SessionsController < Devise::SessionsController
   # end
 
   # POST /resource/sign_in
-  # def create
-  #   super
-  # end
+  def create
+    if User.find_by(email: params[:user][:email]).blocked
```

Comment:
Please do not find user in create method. Wont we be getting current user or something here??
This should not be in action. This should be a before action